src/main/java/org/acra/ACRAConfiguration.java

```diff
@@ -320,6 +321,12 @@ public ACRAConfiguration setResDialogNegativeButtonText(int resId) {
         return this;
     }
 
+    @SuppressWarnings( "unused" )
+    public ACRAConfiguration setReportDialogClass(Class<? extends CrashReportDialog> reportDialogClass) {

```

Comment:
Is there any specific reason to force a subclass of CrashReportDialog? Otherwise specifying it as Class<? extends Activity> would make it a lot more flexible as one would be able to provide a completely different Activity.